cylc/flow/graph_parser.py

```diff
@@ -89,6 +89,8 @@ class GraphParser:
     FAM_TRIG_EXT_ANY = '-any'
     LEN_FAM_TRIG_EXT_ALL = len(FAM_TRIG_EXT_ALL)
     LEN_FAM_TRIG_EXT_ANY = len(FAM_TRIG_EXT_ANY)
+    CONTINUATION_STRS = [ARROW, OP_AND, OP_OR]
+    BAD_STRS = [OP_AND_ERR, OP_OR_ERR]
```

Comment:
CI is happy, docs look good. Great job (and really quick) @wxtim . These bad strings are the only bit I didn't understand. I thought parsec would raise an error when finding these bad strings in the next line (or other invalid values like => after a boolean). Thanks!

Reply:
These represent bad content inside the graph strings. Parsec is (mostly) a generic `item = value` config parser, so to it the entire graph string is just a string.